File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/adt/shared/DTOs/discharge-cancel.dto.ts

export class DischargeCancel_DTO {
  public PatientVisitId: number = null;
  public PatientAdmissionId: number = null;
  public DischargedDate: string = "";
  public CreatedOn: string = "";
  public DischargedBy: number = null;
  public DischargeCancelledBy: number = null;
  public BillingTransactionId: number = null;
  public DischargeCancelNote: string = "";
  public CounterId: number = null;
  public NewBedId: number = null;
  //public DischargeCancel = new DischargeCancel();
}

File: Code/Websites/DanpheEMR/Services/Admission/DTOs/DischargeCancel_DTO.cs
using System.ComponentModel.DataAnnotations.Schema;
using System;
using DanpheEMR.ServerModel;

namespace DanpheEMR.Services.Admission.DTOs
{
    public class DischargeCancel_DTO
    {
        public int? PatientVisitId { get; set; }
        public int? PatientAdmissionId { get; set; }
        public DateTime DischargedDate { get; set; }
        public DateTime? CreatedOn { get; set; }
        public int? DischargedBy { get; set; }
        public int? DischargeCancelledBy { get; set; }
        public int? BillingTransactionId { get; set; }
        public string DischargeCancelNote { get; set; }
        public int CounterId { get; set; }
        public int? NewBedId { get; set; }
        //public DischargeCancelModel DischargeCancel { get; set; }
    }
}
